finishFeature/src/main/java/com/panhuk/finishfeature/FinishFragment.kt

```diff
@@ -0,0 +1,109 @@
+package com.panhuk.finishfeature
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Button
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.fragment.app.Fragment
+import com.panhuk.core.RIGHT_ANSWERS
+import com.panhuk.core.TOTAL_ANSWERS
+import com.panhuk.core.Screen
+import com.panhuk.finishfeature.databinding.FinishFragmentBinding
+
+class FinishFragment : Fragment() {
+  private var _binding: FinishFragmentBinding? = null
+  private val binding get() = _binding!!
+  private var rightAnswers: Int = 0
+  private var totalAnswers: Int = 0
+
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View {
+    _binding = FinishFragmentBinding.inflate(inflater, container, false)
+    return binding.root
+  }
+
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+    arguments?.run {
+      rightAnswers = getInt(RIGHT_ANSWERS)
+      totalAnswers = getInt(TOTAL_ANSWERS)
+    }
+    binding.finishFragment.setContent { CreateFinishFragment() }
+  }
+
+  override fun onDestroyView() {
+    _binding = null
+    super.onDestroyView()
+  }
+
+  @Preview(showBackground = true)
+  @Composable
+  fun CreateFinishFragment() {
```

Comment:
Please, read naming convention of Compose functions [here](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#naming-unit-composable-functions-as-entities).

Reply:
make sure you understand it and apply it to every compose function in this class and in the future.